overview.md

```diff
@@ -1,3 +1,7 @@
+⚠️ <u>**Important notice for version 3.0.0 and above**</u>:
+To make the Checkmarx One plugin for ADO more lightweight, the CLI tool has been removed and now needs to be downloaded separately during use. This change may affect customers who restrict access to whitelisted domains, requiring them to add **download.checkmarx.com** to their whitelist.
```

Comment:
Important notice for version 3.0.0 and above
To make the Checkmarx One plugin for ADO more lightweight, the CLI tool has been removed and now needs to be downloaded separately during use. This change may affect customers who restrict access to whitelisted domains, requiring them to add download.checkmarx.com to their whitelist. 
Customers who are unable to whitelist this domain, can use an older version of the plugin, which is available for download [here](https://github.com/Checkmarx/ast-azure-plugin/releases).